app/src/lib/git/reflog.ts

```diff
@@ -9,7 +9,10 @@ export async function getRecentBranches(
   // "git reflog show" is just an alias for "git log -g --abbrev-commit --pretty=oneline"
   // but by using log we can give it a max number which should prevent us from balling out
   // of control when there's ginormous reflogs around (as in e.g. github/github).
-  const regex = new RegExp(/.*? checkout: moving from .*? to (.*?)$/i)
+  const regex = new RegExp(
+    /.*? (renamed|checkout)(?:: moving from|\s*) (?:refs\/heads\/|\s*)(.*?) to (?:refs\/heads\/|\s*)(.*?)$/i
```

Comment:
This expression can be simplified a bit.
```suggestion
   /.*?\srefs\/heads\/(.*?)\sto\srefs\/heads\/(.*?)$/i
```

If the `\s` don't read well, they can be substituted for literal whitespace characters, but that's really a matter of personal style.

Reply:
@iAmWillShepherd  thanks for the code review.
actually we need both renamed and new checkout (created) branches at the same time, otherwise, the recent branch list will not be complete and there will be only renamed branches under the category

Reply:
Just tested app with `/.*?(renamed|checkout) refs\/heads\/(.*?) to refs\/heads\/(.*?)$/i` and seems to work OK (I ended up subbing `\s` for literal whitespace).